src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.WinRT.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
That is the goal I just assume the WinRT code is going away in 2.1.

Incidentally I'm not sure how important it is to us to have a common binary between unix and Windows. Clearly less so.